test/e2e/framework/test_context.go

```diff
@@ -129,4 +130,5 @@ func RegisterFlags() {
 	flag.BoolVar(&TestContext.GatherMetricsAfterTest, "gather-metrics-at-teardown", false, "If set to true framwork will gather metrics from all components after each test.")
 	flag.StringVar(&TestContext.OutputPrintType, "output-print-type", "hr", "Comma separated list: 'hr' for human readable summaries 'json' for JSON ones.")
 	flag.BoolVar(&TestContext.DumpLogsOnFailure, "dump-logs-on-failure", true, "If set to true test will dump data about the namespace in which test was running.")
+	flag.IntVar(&TestContext.Scale, "scale", 1, "Tests can use this parameter to multiply their load on a cluster, where scale = 1 represents a basic relative load of 1.  This is interpretted differently by different tests.  In general should be used only by experts.")
```

Comment:
@spxtr i think it makes more sense now?  but i miss the `multiply your fury` part